students/Zhengtang_Yang/GridPrinterExercise.py

```diff
@@ -0,0 +1,38 @@
+print('+ - - - - + - - - - +')
+print('|         |         |')
+print('|         |         |')
+print('|         |         |')
+print('|         |         |')
+print('+ - - - - + - - - - +')
+print('|         |         |')
+print('|         |         |')
+print('|         |         |')
+print('|         |         |')
+print('+ - - - - + - - - - +')
+
+def print_grid(n):
+	a1 = '+ '+'- '*((n-1)//2)+'+ '+'- '*((n-1)//2)+'+ '+'\n'
+	a2 = '| '+'  '*((n-1)//2)+'| '+'  '*((n-1)//2)+'| '+'\n'
+	a3 = a1+a2*((n-1)//2)+a1+a2*((n-1)//2)+a1
+	print(a3)
+
+print('print_grid(3)')
+print_grid(3)
+
+print('print_grid(11)')
+print_grid(11)
+
+print('print_grid(15)')
+print_grid(15)
+
+print('print_grid2(n,m)')
+def print_grid2(n,m):
```

Comment:
The goal with grid2 was use the first value to make it a n by n grid with the second value being the size of the box. So an entry of `print_grid2(5,3)` would result in:
```
+ - - - + - - - + - - - + - - - + - - - +
|       |       |       |       |       |
|       |       |       |       |       |
|       |       |       |       |       |
+ - - - + - - - + - - - + - - - + - - - +
|       |       |       |       |       |
|       |       |       |       |       |
|       |       |       |       |       |
+ - - - + - - - + - - - + - - - + - - - +
|       |       |       |       |       |
|       |       |       |       |       |
|       |       |       |       |       |
+ - - - + - - - + - - - + - - - + - - - +
|       |       |       |       |       |
|       |       |       |       |       |
|       |       |       |       |       |
+ - - - + - - - + - - - + - - - + - - - +
|       |       |       |       |       |
|       |       |       |       |       |
|       |       |       |       |       |
+ - - - + - - - + - - - + - - - + - - - +```